docs/ExpectAPI.md

```diff
@@ -79,6 +85,20 @@ declare global {
 }
 ```
 
+If you want to move the typings to a separate file (e.g. `types/jest/index.d.ts`), you may need to an export, e.g.:
+
+```ts
+declare global {
+  namespace jest {
+    interface Matchers<R> {
+      toBeWithinRange(a: number, b: number): R;
+    }
+  }
+}
+
+export {};
+```
```

Comment:
```suggestion
If you prefer to keep the typings in a `.d.ts` declaration file, you may need to include an empty `export {}`.
```

Just thinking out loud. Seems like there is no need to repeat the code again. Or my version is less clear? Feel free to rework it.